### PR TITLE
Add blubber pipeline stanzas

### PR DESCRIPTION
**Pull request by akosiaris**
Add .pipeline/blubber.yaml

Bug: T213193
Change-Id: I80e1ba75ab8054e75cca48609fb140faeb0c1f7d

### CODE REVIEW

**Comment by akosiaris** [comment]
Travis complains because:
```
E: Package 'oracle-java8-set-default' has no installation candidate
```

not sure how to fix it.

File: .pipeline/blubber.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+apt: { packages: [librdkafka++1, librdkafka1] }
+lives:
+  in: /srv/service
+runs:
+  environment: { APP_BASE_PATH: /srv/service }
+
+variants:
+  build:
+    base: docker-registry.wikimedia.org/nodejs-devel
+    apt: { packages: [librdkafka-dev] }
```
Comment (d00rman) [inline .pipeline/blubber.yaml]: For the build step, we need this package to come from our WMF repo.

File: .pipeline/blubber.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    node: { requirements: [package.json]}
+    runs: { environment: { LINK: g++ } }
```
Comment (d00rman) [inline .pipeline/blubber.yaml]: Also needs the `BUILD_LIBRDKAFKA=0` env var set.

**Comment by d00rman** [comment]
cf. [package.json's build stanza](https://github.com/wikimedia/change-propagation/blob/bb78865d392ebdbcfde623fdbc13358bfe0dccaf/package.json#L69-L88) for the build reqs.

File: .pipeline/blubber.yaml (hunk 1 above)
Comment (akosiaris) [reply .pipeline/blubber.yaml]: That happens anyway in the pipeline. In the images, `apt.wikimedia.org` is always preferred, same way as in production

**Comment by akosiaris** [comment]
The repo comment did not need addressing, I did address the ENV var one.

**Pull request merged**